manager/controllers/app/blueprint_controller.go

```diff
@@ -273,8 +274,18 @@ func (r *BlueprintReconciler) checkResourceStatus(res *unstructured.Unstructured
 		return corev1.ConditionUnknown, ""
 	}
 	if expected == nil {
-		// TODO: use kstatus to compute the status of the resources for them the expected results have not been specified
-		return corev1.ConditionTrue, ""
+		// use kstatus to compute the status of the resources for them the expected results have not been specified
+		// Current status of a deployed release indicates that the resource has been successfully reconciled
+		// Failed status indicates a failure
+		computedResult, _ := kstatus.Compute(res)
```

Comment:
If `_` is an ignored error then please don't ignore

Reply:
Done.